app/mappers/userMapper.js

```diff
@@ -3,11 +3,12 @@
  * Added in order to avoid return password as response.
  */
 class UserResponse {
-  constructor({ id, firstName, lastName, email, createdAt, updatedAt } = {}) {
+  constructor({ id, firstName, lastName, email, role, createdAt, updatedAt } = {}) {
     this.id = id;
     this.first_name = firstName;
     this.last_name = lastName;
     this.email = email;
+    this.role = role;
     this.created_at = createdAt;
     this.updated_at = updatedAt;
   }
```

Comment:
The same I have commented in other PR, make this a anon/lambda/arrow function instead of a class